custom_components/kia_uvo/Vehicle.py

```diff
@@ -73,6 +72,9 @@ async def update(self):
                 _LOGGER.debug(
                     f"zero battery api error, force_update started to correct data"
                 )
+                self.vehicle_data["vehicleStatus"]["evStatus"] = {}
```

Comment:
Not sure how this will impact existing sensors, they will probably generate unavailable, right?